### PR TITLE
[Decomposition] _unsafe_view

### PR DESCRIPTION
**Pull request by salilsdesai**
Summary:
Decomp already exists so just add it to core_aten_decompositions

https://www.internalfb.com/code/fbsource/[9d5eabd7b213d1a356d4e7bb400355d574ea924b]/fbcode/caffe2/torch/_decomp/decompositions.py?lines=3091

Differential Revision: D48619079

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov

### CODE REVIEW

**Review by larryliu0820 — APPROVED** [review]

**Review by SherlockNoMad — APPROVED** [review]
Great.

File: torch/_inductor/decomposition.py [hunk 1]
```diff
@@ -71,6 +71,7 @@
 decomps_to_exclude = [
     aten._unsafe_index,
     aten._scaled_dot_product_flash_attention.default,  # See comments in torch/_decomp/decompositions.py
+    aten._unsafe_view,
```
Comment (peterbell10) [inline torch/_inductor/decomposition.py:74]: Why are you not decomposing it in inductor?

Comment (salilsdesai) [reply torch/_inductor/decomposition.py:74]: Inductor has a lowering for this:

https://www.internalfb.com/code/fbsource/[ba02efe6ad091b414cb786eaf9d6172fe887b857]/xplat/caffe2/torch/_inductor/lowering.py

According to Sherlock, 

> Inductor has lowering for this: implying inductor would want this op preserved in graph.
> Please exclude this from inductor decomp.

https://github.com/pytorch/pytorch/pull/108546?fbclid=IwAR0PhDU5AAJRiFPRcIQyU9r-rPd2ARaF2T41xMYv-YNTtkWfRIjdm98L3xo#discussion_r1320206370

Comment (eellison) [reply torch/_inductor/decomposition.py:73]: This makes pattern matching more difficult, we should be including it.

**Pull request closed without merging**